ui/app/adapters/ldap/role.js

```diff
@@ -3,46 +3,92 @@
  * SPDX-License-Identifier: BUSL-1.1
  */
 
-import NamedPathAdapter from 'vault/adapters/named-path';
+import ApplicationAdapter from 'vault/adapters/application';
```

Comment:
we can't use `name` as the primaryKey because static and dynamic roles can technically have the same name